src/Task.ts

```diff
@@ -263,18 +266,32 @@ export class Task {
         return li;
     }
 
-    public toString(): string {
-        const recurrenceRule: string = this.recurrenceRule
-            ? ` 🔁 ${this.recurrenceRule.toText()}`
-            : '';
-        const dueDate: string = this.dueDate
-            ? ` 📅 ${this.dueDate.format(Task.dateFormat)}`
-            : '';
-        const doneDate: string = this.doneDate
-            ? ` ✅ ${this.doneDate.format(Task.dateFormat)}`
-            : '';
-
-        return `${this.description}${recurrenceRule}${dueDate}${doneDate}${this.blockLink}`;
+    public toString(layoutOptions?: LayoutOptions): string {
```

Comment:
Personally I would prefer `toString({layoutOptions}: {layoutOptions?: LayoutOptions})`, but it’s ok if you leave it as is. It’s especially fine the way it is when there is only a single argument.

Reply:
I left it as it is since I did not find out how to make it optional like this.